integreat_cms/cms/templates/events/event_form_sidebar/icon_box.html

```diff
@@ -11,5 +11,17 @@
 {% endblock collapsible_box_title %}
 {% block collapsible_box_content %}
     <label class="mt-0" for="{{ event_form.icon.id_for_label }}">{{ event_form.icon.label }}</label>
-    {% render_field event_form.icon label=event_form.icon.label %}
+    {% if perms.cms.change_event %}
+        {% render_field event_form.icon label=event_form.icon.label %}
+    {% else %}
+        {% if event_form.instance.icon.url %}
+            {# djlint:off H006 #}
+            <img src="{{ event_form.instance.icon.url }}"
+                 class="h-40 m-auto"
+                 alt="{{ event_form.instance.icon.url }}"/>
+            {# djlint:on #}
+        {% else %}
+            {% translate "No icon is set" %}
+        {% endif %}
+    {% endif %}
```

Comment:
```suggestion
    {% render_field event_form.icon label=event_form.icon.label %}
```
I think it would be more in line with Django's design principles if we always render the form field here - and just don't show the edit buttons in the template of the icon widget:

Wrap the following block with `{% if not widget.attrs.disabled %}`:

https://github.com/digitalfabrik/integreat-cms/blob/3a27a1732671c41c1d9854f1a293798837616459/integreat_cms/cms/templates/icon_widget.html#L17-L45